src/listpack.c

```diff
@@ -1215,7 +1215,12 @@ int lpValidateIntegrity(unsigned char *lp, size_t size, int deep,
 }
 
 /* Compare entry pointer to by 'p' with string 's' of length 'slen'.
- * Return 1 if equal. */
+ * Return 1 if equal.
+ *
+ * Note that the entry 'p' may be integer-encoded. In such a case we
+ * use lpStringToInt64() to get an integer representation of the
+ * string 's' and compare to 'sval', it's much faster than convert
+ * integer to string and comparing. */
```

Comment:
i think that comment belongs to the body of the function (next to the code), it doesn't explain how to use the function or what it does, so no need to have it at the top

Reply:
Done.